lib/anima/attribute.rb

```diff
@@ -42,7 +42,7 @@ def initialize(name)
     #
     def load(object, attributes)
       value = attributes.fetch(name) do |attribute_name|
-        fail Error::Missing.new(object, attribute_name)
+        return
```

Comment:
*same semantics for the anima public API. `Attribute#load` is private. I'm fine to raise a KeyError here from `Hash#fetch`, because it should never happen.